trieste/acquisition/_optimizer.py

```diff
@@ -62,11 +63,16 @@ def _box(space: Box, target_func: TensorMapping) -> QueryPoints:
     initial_point = optimize(trial_search_space, target_func)
 
     bijector = tfp.bijectors.Sigmoid(low=space.lower, high=space.upper)
-    variable = tf.Variable(bijector.inverse(initial_point))
 
-    def _objective() -> tf.Tensor:
-        return -target_func(bijector.forward(variable))
+    def _objective(x: tf.Variable) -> tf.Tensor:
+        return -target_func(bijector.forward(x))[..., 0]
```

Comment:
If you're expecting the last dimension to be 1, the following makes that assumption clear in code:
```suggestion
        return -tf.squeeze(target_func(bijector.forward(x)), axis=-1)
```

Reply:
yeah while the original is slightly more readable imo, `tf.squeeze` is safer e.g. in case the rest of the code changes